rad/src/operators/array.rs

```diff
@@ -49,14 +49,16 @@ pub fn get(input: &RadonArray, args: &[Value]) -> Result<RadonTypes, RadError> {
         return Err(wrong_args());
     }
 
-    let not_found = |index: i32| RadError::ArrayIndexNotFound { index };
+    let not_found = |index: usize| RadError::ArrayIndexNotFound {
+        index: i32::try_from(index).unwrap(),
+    };
 
     let arg = args[0].to_owned();
-    let index = from_value::<i32>(arg).map_err(|_| wrong_args())?;
+    let index = from_value::<usize>(arg).map_err(|_| wrong_args())?;
```

Comment:
Good find, it doesn't make sense to index an array with negative values. This has some problems, I opened issue #1087 in order to further discuss that. In the meanwhile we can just return `not_found` error on negative values:

```suggestion
    let index = from_value::<i32>(arg).map_err(|_| wrong_args())?;
    let index = usize::try_from(index).map_err(|_| not_found(index))?;
```